Source/WebKit/UIProcess/Automation/win/WebAutomationSessionWin.cpp

```diff
@@ -349,7 +350,7 @@ OptionSet<WebEventModifier> WebAutomationSession::platformWebModifiersFromRaw(We
 void WebAutomationSession::platformSimulateKeySequence(WebPageProxy& page, const String& keySequence)
 {
     // https://www.w3.org/TR/uievents-code/#key-alphanumeric-writing-system
-    String keyIdentifier = "Key"_s + keySequence.convertToASCIIUppercase();
+    auto keyIdentifier = makeString("Key"_s, keySequence.convertToASCIIUppercase());
```

Comment:
This should use `uppercase()` to avoid creating an intermediate `StringImpl`. Or that might not be the exact name.